### ISSUE TITLE
Import coala.io contributors functionality

### ISSUE THREAD

**Issue by jayvdb** [coala/community#32]
https://coala.io/#/getinvolved is part of https://github.com/coala/landing-frontend , and it uses data collated from the webservice provided by https://gitlab.com/coala/landing .

The same backend data should be imported into a data model in this repo, with a similar UI, but UI is not so important.

Note: The data from the webservice is a bit out of date.  Not a problem for this task.

**Comment by sks444** [comment]
@jayvdb, One way to do it is to get all the contributor's data into a JSON file and load it into a `Contributor` model by `loaddata` but for this, we need a database and we are not using one here.  So how should we do it another way?

**Comment by ashwini0529** [comment]
@sks444 contributors dataset is something that will be updated. So I would suggest to work with some sort of API that could fetch contributors and show them.

**Comment by sks444** [comment]
@ashwini0529,
 

> contributors dataset is something that will be updated. So I would suggest to work with some sort of API that could fetch contributors and show them.

Yes, that is what I was thinking. Thank you :)
BTW, how we are fetching the data in https://github.com/coala/landing-frontend from the backend?

**Comment by sks444** [comment]
I am thinking about making an API which scrapes the data from https://coala.io/#/getinvolved and posts it in the Contributor model.

**Comment by ashwini0529** [comment]
@sks444 why do you need to store the data?

**Comment by sks444** [comment]
@ashwini0529, Because that is what the issue is about, we need to import the data into a data model, we are not showing it here, yet.

**Comment by sks444** [comment]
@ashwini0529, Oh, I have got it now. We don't have to do all this, we just have to implement what already has been done in https://gitlab.com/coala/landing/tree/master/backend/org for this repo, as @jayvdb, had said earlier, this task is a 'copy and paste.'
@yashraj-Dhanuka, are you working on it or should I create a PR?

**Comment by yashraj-Dhanuka** [comment]
@sks444 Actually my exams were going on, so i could not get time on this.
I will work on it today , Thanks for the help bro.

**Comment by jayvdb** [comment]
> we just have to implement what already has been done in gitlab.com/coala/landing/tree/master/backend/org for this repo

Not quite.  This repository should not re-create all that data.  It is time consuming.  This repository should fetch that data from the *webservice* provided by http://gitlab.com/coala/landing .

To do this task, you must run http://gitlab.com/coala/landing on your machine to use for testing, populate the data in that app , and then use its webservice (web api on your machine) to populate a new data model for this repository.

To put it another way, each time this "community" project is built, the apps in this repository should be able to very easily obtain data from the same webservice that is behind http://coala.io/ .

**Comment by jayvdb** [comment]
I've closed the PR as it appears to be stalled in the wrong direction.
